pytorch_lightning/utilities/warnings.py

```diff
@@ -41,14 +41,11 @@ class PossibleUserWarning(UserWarning):
     """Warnings that could be false positives."""
 
 
-class LightningDeprecationWarning(DeprecationWarning):
+class LightningDeprecationWarning(FutureWarning):
```

Comment:
@puhuk Could you remove this as we don't need our own warning? 
```suggestion
```

Also, we can get rid of the following line:
https://github.com/PyTorchLightning/pytorch-lightning/blob/5ad5ba54c0c477546d21daf75ac7b4748d7963a7/setup.cfg#L29